### PR TITLE
numerous fixes to samples to bring them inline with Reunion 0.5 prerelease and GA

### PR DESCRIPTION
**Pull request by Scottj1s**
Samples have been updated to be consistent with changes in the Reunion VSIX project templates
update package references to Reunion 0.5 prerelease
remove obsolete AddReunionPackageDependency.ps1
rationalize TPV to eliminate SupportedOSPlatform warnings
remove cruft from wapproj files, which will enable upgrading Reunion package references
remove unnecessary VCRT forwarders
remove experimental Suspending event to support Reunion 0.5 GA

### CODE REVIEW

**Review by axelandrejs — APPROVED** [review]

File: MrtCore/winui_desktop_packaged_app/winui_desktop_packaged_app (Package)/winui_desktop_packaged_app (Package).wapproj [hunk 1]
```diff
@@ -37,10 +37,9 @@
   <PropertyGroup>
     <ProjectGuid>28b258ca-5ecc-41ee-a5b0-c2f24f3a4984</ProjectGuid>
     <TargetPlatformVersion>10.0.19041.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+    <TargetPlatformMinVersion>10.0.18362.0</TargetPlatformMinVersion>
```
Comment (rohanp-msft) [inline MrtCore/winui_desktop_packaged_app/winui_desktop_packaged_app (Package)/winui_desktop_packaged_app (Package).wapproj:40]: >18362 [](start = 35, length = 5)

Curious Scott, why is this needed or good to do?

File: MrtCore/winui_desktop_packaged_app_cpp/winui_desktop_packaged_app_cpp/winui_desktop_packaged_app_cpp.vcxproj [hunk 2]
```diff
@@ -18,11 +18,11 @@
     <AppxPackage>false</AppxPackage>
     <ApplicationType>Windows Store</ApplicationType>
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
-    <TargetPlatformMinVersion>10.0.18362.0</TargetPlatformMinVersion>
+    <TargetPlatformMinVersion>10.0.19041.0</TargetPlatformMinVersion>
```
Comment (rohanp-msft) [inline MrtCore/winui_desktop_packaged_app_cpp/winui_desktop_packaged_app_cpp/winui_desktop_packaged_app_cpp.vcxproj:21]: >10.0.19041.0 [](start = 30, length = 12)

This is 10.0.17763.0 in the wapproj. Would it be good to match that? Do we need to have such a high min version?

File: MrtCore/winui_desktop_packaged_app/winui_desktop_packaged_app (Package)/winui_desktop_packaged_app (Package).wapproj (hunk 1 above)
Comment (Scottj1s) [reply MrtCore/winui_desktop_packaged_app/winui_desktop_packaged_app (Package)/winui_desktop_packaged_app (Package).wapproj:40]: I changed it to match winui_class_lib, to eliminate the SupportedOSPlatform warning.  winui_class_lib does not build with 17763, so I had to leave it at 18362, but for consistency all minvers should probably also be 18362.

File: MrtCore/winui_desktop_packaged_app_cpp/winui_desktop_packaged_app_cpp/winui_desktop_packaged_app_cpp.vcxproj (hunk 2 above)
Comment (Scottj1s) [reply MrtCore/winui_desktop_packaged_app_cpp/winui_desktop_packaged_app_cpp/winui_desktop_packaged_app_cpp.vcxproj:21]: a mistake - reverted to 18362 to match winui_class_lib
